src/lib/components/chat/MessageInput/VoiceRecording.svelte

```
@@ -2,7 +2,7 @@
 	import { toast } from 'svelte-sonner';
 	import { createEventDispatcher, tick, getContext, onMount, onDestroy } from 'svelte';
 	import { config, settings } from '$lib/stores';
-	import { blobToFile, calculateSHA256, findWordIndices } from '$lib/utils';
+	import { blobToFile, calculateSHA256, extractCurlyBraceWords } from '$lib/utils';
```

ESLint

[error] 'calculateSHA256' is defined but never used.

(@typescript-eslint/no-unused-vars)